client/me/profile-links/index.jsx

```diff
@@ -175,33 +160,39 @@ module.exports = React.createClass( {
 					? this.renderProfileLinks()
 					: this.renderNoProfileLinks()
 				}
+			</div>
+		);
+	},
 
-				{
-					'wordpress' === this.state.showingForm
-					? (
-						<ProfileLinksAddWordPress
-							userProfileLinks={ this.props.userProfileLinks }
-							onSuccess={ this.hideForms }
-							onCancel={ this.hideForms }
-						/>
-					)
-					: null
-				}
-
-				{
-					'other' === this.state.showingForm
-					? (
-						<ProfileLinksAddOther
-							userProfileLinks={ this.props.userProfileLinks }
-							onSuccess={ this.hideForms }
-							onCancel={ this.hideForms }
-						/>
-					)
-					: null
-				}
-
-				{ ! this.state.showingForm ? this.renderFormVisibilityControls() : null }
-		</div>
+	render: function() {
+		return(
+			<div>
+				<SectionHeader label={ this.translate( 'Profile Links' ) }>
```

Comment:
I had to do this for settings as well, and it made me think that it'd make more sense semantically for SectionHeading to be within the Card instead of having to add an extra div wrapper. What do you think?